.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+contact-links:
```

Comment:
this should be an underscore. totally on me... 🤦

Reply:
I just realized that right now...